airbyte-integrations/connectors/destination-bigquery/src/test-integration/java/io/airbyte/integrations/destination/bigquery/BigQueryDestinationTest.java

```diff
@@ -140,15 +135,21 @@ void setup(TestInfo info) throws IOException {
         .build()
         .getService();
 
-    final String datasetId = "airbyte_tests_" + RandomStringUtils.randomAlphanumeric(8);
+    String datesetId = "airbyte_tests_" + RandomStringUtils.randomAlphanumeric(8);
```

Comment:
This shift here is because we generate a new dataset id for each test and we want to pass this into the catalog as the namespace.

Reply:
You didn't need to remove the `final` though?

Reply:
oh yes. thanks!